Add IDL harness tests for motion sensors

Honry opened the pull request:
- Add IDL harness tests for Accelerometer, Gyroscope, Magnetometer
 - Add @Honry as test owner

tobie commented on accelerometer/idlharness.https.html:
```diff
@@ -0,0 +1,113 @@
+[Constructor(optional AccelerometerOptions accelerometerOptions)]
+interface Accelerometer : Sensor {
+  readonly attribute AccelerometerReading? reading;
+  readonly attribute boolean includesGravity;
```
`includesGravity` has still not been fixed in the spec?

Honry replied:
No, it's not yet, see the latest [ED](https://w3c.github.io/accelerometer/#accelerometer-interface).

Honry replied:
@tobie, can we merge it at first? As more new tests for motion sensors depend on this PR.
Once the API changes, I can help update tests immediately.

tobie replied:
I don't have an opinion on this, tbh.

Honry committed:
Add IDL harness tests for motion sensors

 - Add IDL harness tests for Accelerometer, Gyroscope, Magnetometer
 - Add @Honry as test owner

Honry commented:
Many thanks @tobie for the review, @dontcallmedom, could you help merge this PR?

tobie commented:
Top directories need to match shortnames for the tooling to work. These should be in three directories.

tobie commented:
@Honry: for the record, I did not review those tests. I just commented on one issue that jumped at me after a cursory glance.

At the very least, you want to split those tests up and put them in separate folders as they're separate specs.

Honry commented:
@tobie, never mind. 😄
